bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/HDPowerViewWebTargets.java

```diff
@@ -75,6 +76,7 @@
  */
 @NonNullByDefault
 public class HDPowerViewWebTargets {
+    private static final int REQUEST_TIMEOUT_MS = 10000;
```

Comment:
I'm almost sure this value is too low. Some call will query the shades directly (i.e. HTTP call to hub -> RF to shade) for refreshing battery level, signal strength etc. Some of these calls are quite slow.

Reply:
Doubled it.

Reply:
I just did a quick test of one specific API call I remember to be quite slow:
```
2023-08-26 23:40:11.059 [INFO ] [openhab.event.ItemCommandEvent      ] - Item 'Blind10_SignalStrength' received command REFRESH
2023-08-26 23:40:22.230 [INFO ] [rnal.handler.HDPowerViewShadeHandler] - No data from shade 24076 survey
```

I think 30 seconds would be on the safer side.

Slow calls are slow, and we'll have to wait for them. Or we would need to refactor to async calls. Fast calls shouldn't usually be a problem, even if timeout is 5 minutes, since they will only block for a very short time.

I'm now wondering if it would be better not to use the common HTTP client for sync call scenarios which are known to be slow - like this one and Bosch Indego.

Reply:
I'll set it to 30secs.
Don't think it will be a problem for the common client. But time will tell, if so we can create a PR easily.